apps/omg_watcher/test/omg_watcher/integration/block_getter_test.exs

```diff
@@ -133,11 +131,18 @@ defmodule OMG.Watcher.Integration.BlockGetterTest do
     block_with_incorrect_hash = %{OMG.Block.hashed_txs_at([], 1000) | hash: different_hash}
 
     # from now on the child chain server is broken until end of test
-    BadChildChainServer.prepare_route_to_inject_bad_block(
```

Comment:
the setting was forced via shared state (application setting) which is overall discouraged approach